src/localeStorageManager.js

```diff
@@ -18,7 +31,13 @@ export const localSettings = {
 
 export const getIsDevMode = () => {
   const settings = localSettings.fetch();
-  return settings.devMode ?? false;
+  return settings[localSettingsKeys.isDevMode] ?? true;
```

Comment:
If the mode is not registered, we assume the value is `false` otherwise everyone would be in dev mode during their first visit

```suggestion
  return settings[localSettingsKeys.isDevMode] ?? false;
```

Reply:
My bad, fixed now